services/minespace-web/src/styles/overrides/antd-overrides.scss

```diff
@@ -436,6 +436,14 @@ tr.ant-table-expanded-row {
   }
 }
 
+.ant-radio-group {
```

Comment:
Did we ensure that this override did not have unintended effects on other components that use this class?

Reply:
I checked for any unwanted effects on all the radio groups I could find and they seem to be behaving normally.  This modal is the only one using longer strings of text attached to the buttons, and without this it was overrunning it's container rather than wrapping.